src/gausskernel/storage/access/k2/pg_gate_api.cpp

```diff
@@ -820,6 +820,8 @@ K2PgStatus PgGate_NewSelect(K2PgOid database_oid,
     }
     (*handle)->secondarySchema = secondarySchema;
 
+    (*handle)->maxParallelReads = k2pg::Config().get<uint32_t>("pggate.max_parallel_reads", 5);
```

Comment:
This is going to parse the config file on every call. I think we should make the config in the TxMgr public and then you can access it here via TxMgr.config